<launch>
  <node pkg="localization_monitor" type="localization_monitor" name="localization_monitor" output="screen">

	<param name="uncertainty_thres" value="0.15"/>
	<param name="score_thres" value="0.5"/>
	<param name="inflation_thres" value="0.2"/>
	<param name="max_lim" value="3.0"/>
	<param name="min_valid_points" value="20"/>

	<param name="ang_thres" value="15.0"/>
	<param name="dis_thres" value="0.2"/>
  	<param name="dis_thres_check" value="1.5"/>
	<param name="rate_thres" value="0.5"/>	

  </node>
</launch>

